descriptors/ModuleDescriptor-template.json

```diff
@@ -111,6 +111,15 @@
             "search.bibframe.collection.get"
           ]
         },
+        {
```

Comment:
Let us add this API to README file
| GET    | `/search/bibframe/authorities `            | Search linked data graph authority resource descriptions        |